src/openforms/js/components/admin/form_design/registrations/index.js

```diff
@@ -10,4 +12,15 @@ export const BACKEND_OPTIONS_FORMS = {
     form: CamundaOptionsForm,
     onStepEdit: onStepEdit,
   },
+  email: {
+    uiSchema: {
+      emailContentTemplateText: {
+        'ui:widget': 'textarea',
+        'ui:options': {
+          rows: 5,
```

Comment:
I would have liked to overwrite the `cols` too, but it doesn't look like there is that option: https://github.com/rjsf-team/react-jsonschema-form/blob/main/packages/core/src/components/widgets/TextareaWidget.tsx#L47

If we really want it, i will make a custom widget also for the textarea

Reply:
can't you do this with CSS and give it a particular width?